src/components/Searchbar.js

```diff
@@ -1,15 +1,161 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
 import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
 import constants from '../styling/Constants';
 import { TextInput, DateInput, StyledButton } from '../components';
+import { useHistory } from 'react-router-dom';
+
+function getInputVal(wrapId) {
+    return document.getElementById(wrapId)?.getElementsByTagName('input')[0]?.value;
+}
+
+function getLocal(key) {
+    return window.localStorage.getItem(key);
+}
+
+function saveLocalSearch() {
+    window.localStorage.setItem('location', getInputVal('location-input-wrap'));
+
+    let date = getInputVal('date-input-wrap');
+    if (date) date = new Date(date);
+    window.localStorage.setItem('dateFrom', date);
```

Comment:
We can probably use indexDB or cache instead of localStorage if it's a small thing - can you look into it? We can look into libraries if need be.

Reply:
localStorage is an outdated way of storing stuff